Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment-Legacy.ps1

```diff
@@ -0,0 +1,134 @@
+param
```

Comment:
This won't work. You're calling `Utility.ps1` and `AzureUtility.ps1` which you've updated to the new lib (calls `Get-VstsLocString` now).